lib/matplotlib/ticker.py

```diff
@@ -1058,9 +1061,27 @@ class EngFormatter(Formatter):
          24: "Y"
     }
 
-    def __init__(self, unit="", places=None):
+    def __init__(self, unit="", places=None, space_sep=True):
+        """ Parameters
```

Comment:
I think this should be on the next line; I'm not sure the leading space trimming will work properly like this.